partials/maven-bom-next.adoc

```diff
@@ -0,0 +1,14 @@
+[source,xml]
+----
+<dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>com.ibm.bamoe</groupId>         
+            <artifactId>bamoe-bom</artifactId> 
+            <version><!-- See Version Mapping Table --></version> 
```

Comment:
can we use some ascidoc link for that table?

Reply:
Yeah, an `xref` link should work.

Reply:
Actually, it doesn't without having to escape all the XML :(

Reply:
then keep it as it is, or add link outside code snippet? do not know, just idea